extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -147,7 +147,12 @@ export class WorkspaceService implements IWorkspaceService {
 		// remove excluded projects specified in workspace file
 		this.excludedProjects = this.getWorkspaceConfigurationValue<string[]>(ExcludedProjectsConfigurationName);
 		this.openedProjects = this.openedProjects.filter(project => !this.excludedProjects?.find(excludedProject => excludedProject === vscode.workspace.asRelativePath(project)));
-
+		// Sort the projects based on the project name extracted from the path property
+		this.openedProjects = this.openedProjects.sort(function (a, b) {
```

Comment:
Does typescript/Node have a data structure that is similar to a SortedList so we don't have to resort it each time? Might not be necessary given the usage but could be good if there's something existing we can leverage.

Reply:
Not that I was able to find with a quick search. =/  Data structures that we take for granted in .NET don't really exist over here.

Reply:
Yup, TypeScript does not have a built-in sorted list kind of data structure.